includes/admin/meta-boxes/views/html-order-items.php

```diff
@@ -253,7 +253,14 @@
 		<?php if ( 'yes' === get_option( 'woocommerce_manage_stock' ) ) : ?>
 			<tr>
 				<td class="label"><label for="restock_refunded_items"><?php esc_html_e( 'Restock refunded items', 'woocommerce' ); ?>:</label></td>
-				<td class="total"><input type="checkbox" id="restock_refunded_items" name="restock_refunded_items" checked="checked" /></td>
+				<?php
+				if ( apply_filter( 'woocommerce_restock_refunded_items', false ) ) {
```

Comment:
`Fatal error: Uncaught Error: Call to undefined function apply_filter() in /srv/www/woodev/wp-content/plugins/woocommerce/includes/admin/meta-boxes/views/html-order-items.php:257 ` 

(You want `apply_filters`)